diffsync/helpers.py

```diff
@@ -396,7 +398,7 @@ def sync_diff_element(self, element: DiffElement, parent_model: Optional["DiffSy
 
         self.incr_elements_processed()
 
-        if not natural_deletion_order:
+        if not natural_deletion_order or self.action is None:
```

Comment:
when `self.action` is None?

Reply:
I guess it needs a comment - imagine we have a parent and a child and the parent has no changes, then its action will be `None`. This means that without this change the children will not be processed if the parent doesn't have any changes, even if the children do.

Reply:
ok, understood

Reply:
Just comparing against the case above (`if natural_deletion_order and self.action == DiffSyncActions.DELETE and not skip_children:`), should this be the inverse, i.e `if not natural_deletion_order or self.action != DiffSyncActions.DELETE`?